File: mongo-api/models/UserNotify.js
const mongoose = require('mongoose');


const userNotifySchema = new mongoose.Schema({
    userId: { type: String, required: true, unique: true },
    fcmToken: String,
    updatedAt: { type: Date, default: Date.now }
});
module.exports = mongoose.model('UserNotify', userNotifySchema);

File: app/src/main/java/com/thien/smart_planner_project/model/UserNotify.java
package com.thien.smart_planner_project.model;

public class UserNotify {
    private String userId;
    private String fcmToken;

    public UserNotify(String userId, String fcmToken) {
        this.userId = userId;
        this.fcmToken = fcmToken;
    }

    public String getUserId() {
        return userId;
    }

    public void setUserId(String userId) {
        this.userId = userId;
    }

    public String getFcmToken() {
        return fcmToken;
    }

    public void setFcmToken(String fcmToken) {
        this.fcmToken = fcmToken;
    }
}